### PR TITLE
Add and Edit Translations page to GDS

### PR DESCRIPTION
**Pull request by farahTW**
This PR transits edit and new translations pages for an organisation to govuk design system.
It does the following:

- Duplicated the existing edit html page to legacy and updated the controller.
- Added new and edit page for translation using GOV UK design system.
- Fixed the controller test and feature tests.
- Corrected the existing featured link partial file for url id field.

**Screen shots:**
image
image

**Trello:**
https://trello.com/c/9QoFofXu/275-add-edit-translations

### CODE REVIEW

File: app/controllers/admin/organisation_translations_controller.rb [hunk 1]
```diff
@@ -20,6 +20,7 @@ def get_layout
   end
 
   def create_redirect_path
+    binding.pry
```
Comment (davidgisbey) [inline app/controllers/admin/organisation_translations_controller.rb]: can we clean this up from the commit history please

File: app/controllers/admin/organisation_translations_controller.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@ def index
 
   def get_layout
     design_system_actions = %w[confirm_destroy]
-    design_system_actions += %w[index] if preview_design_system?(next_release: false)
+    design_system_actions += %w[index edit] if preview_design_system?(next_release: false)
```
Comment (davidgisbey) [inline app/controllers/admin/organisation_translations_controller.rb:13]: this needs update to 

```
    design_system_actions += %w[index edit] if preview_design_system?(next_release: false)

```

or you get this on a failed save

image

File: app/controllers/admin/organisation_translations_controller.rb (hunk 1 above)
Comment (farahTW) [reply app/controllers/admin/organisation_translations_controller.rb]: sure sorry missed it ..

**Review by davidgisbey — APPROVED** [review]
lgtm

**Pull request merged**